README.md

```diff
@@ -46,9 +46,12 @@ Before you install DAHLIA, your system should have the following:
 
 ## Running Tests
 
-Run:
+To run ruby tests:
 - `rake spec`
-- `rake jasmine:ci`
+
+To run javascript unit tests:
+- `rake jasmine:ci` to run in terminal
+- `rake jasmine` to then run tests interactively at http://localhost:8888/
```

Comment:
took me a while to figure out I could do this and then see console logs, so I added it to the README